### PR TITLE
[MAP] Some QoL features for Station 2

### PR DESCRIPTION
**Pull request by HzDonut**
Basically:

1. Adds a crematorium to the medical morgue for disposing of accidentally printed bodies (discussed it with medical a while back, they approved of it at the time). It's in the southeast corner, worked fine during local testing. Debated between adding it to the chapel or medical, decided on medical since we rarely have chaplains and medical needs to use it the most anyways. By the way, it doesn't glow while burning bodies and likes to lock up for a weird amount of time when you ignite without a corpse inside. If anyone experienced with code wants to try to fix that, i'd love it. I'd offer myself but icon_state updates are my bane right now.
2. Adds a hand labeller to explorer prep on the rack. Now you can label your locker so people don't loot the hell out of it (hopefully).
3. Adds a piano to maintenance. I know people bitch at me every single time i discuss adding instruments, but for pete's sake the code got improved and you can just mute instruments if you don't wanna hear it. Stop whining and let people have their fun, for god's sake.

Images:
https://gyazo.com/81b8e10c8bcc27b03f8a32a090e895d3 - Explorer prep
https://gyazo.com/c85053dccbe991392f1cd0b3cd3a1167 - Morgue
https://gyazo.com/2faec21a7d5f72df08ce85b0e9b0c477 - Maint piano

Hopefully working this time.

### CODE REVIEW

**Comment by HzDonut** [comment]
The morgue button got re-adjusted to look a bit better, that's the only difference between this version and the original.

File: maps/tether/tether-06-station2.dmm [hunk 1]
```diff
@@ -4449,10 +4431,12 @@
 /turf/simulated/floor/tiled,
 /area/tether/exploration/crew)
 "gC" = (
-/obj/effect/decal/cleanable/blood,
-/obj/item/weapon/paper,
-/turf/simulated/floor,
-/area/maintenance/station/sec_lower)
+/obj/item/weapon/stool/padded,
+/obj/item/weapon/paper{
+	step_y = 31
```
Comment (TheFurryFeline) [inline maps/tether/tether-06-station2.dmm]: Here's where the first step_xy is.

File: maps/tether/tether-06-station2.dmm [hunk 2]
```diff
@@ -7758,29 +7742,29 @@
...
+	dir = 5
 	},
-/obj/machinery/camera/network/exploration,
-/obj/structure/table/rack/shelf,
-/obj/item/device/multitool/tether_buffered{
-	pixel_y = 2
+/obj/machinery/atmospherics/unary/vent_scrubber/on,
+/obj/structure/table/steel,
+/obj/item/weapon/paper{
+	step_y = 31
```
Comment (TheFurryFeline) [inline maps/tether/tether-06-station2.dmm]: Here's another.

File: maps/tether/tether-06-station2.dmm [hunk 3]
```diff
@@ -15400,27 +15384,27 @@
 /turf/simulated/floor/tiled/techfloor,
 /area/medical/morgue)
 "xy" = (
-/obj/structure/cable/green{
-	d1 = 1;
-	d2 = 8;
-	icon_state = "1-8"
+/obj/effect/decal/cleanable/blood,
+/obj/item/weapon/paper{
+	step_y = 31
```
Comment (TheFurryFeline) [inline maps/tether/tether-06-station2.dmm]: And so on....

**Comment by HzDonut** [comment]
what is this step crap

**Comment by Heroman3003** [comment]
Use pixel_x and pixel_y. step_'s literally break the game.

**Comment by HzDonut** [comment]
I don't even know what you're talking about right now.

**Comment by HzDonut** [comment]
Oh for god's sake now travis is mad again, i hate github so much

**Comment by HzDonut** [comment]
Everything broke, need to make a new PR again since I apparently needed to use "branches" for multiple PRs.

**Pull request closed without merging**